codeforamerica/vita-min
# Delete CTC deprecated actions

## Link to pivotal/JIRA issue
- e.g.[link](https://codeforamerica.atlassian.net/browse/GYR1-7?atlOrigin=eyJpIjoiMjYyNTdhMTQyNTc3NGE2OWFiMGM5OWExNmQ2MmYzZDUiLCJwIjoiaiJ9)

## Is PM acceptance required?

## What was done?
- Removed GetCTC hub pages and tools under the Admin Tools -> Deprecated Actions section
- deleted pages, routes, forms and test files that are related to the following pages: CTC client verification, efile dashboard, verifications, fraud indicators and efile errors

## How to test?
1. Go to the /hub/admin_tools page using the heroku environment
2. Scroll to the bottom
3. Check if Deprecated Actions section is gone
4. Or check if any of these pages still exist: [client verification](https://gyr-review-app-4613-162a6ccee5f3.herokuapp.com/en/hub/verifications), [efile dashboard](https://gyr-review-app-4613-162a6ccee5f3.herokuapp.com/en/hub/verifications), [efile errors](https://gyr-review-app-4613-162a6ccee5f3.herokuapp.com/en/hub/errors), [fraud indicators](https://gyr-review-app-4613-162a6ccee5f3.herokuapp.com/en/hub/fraud-indicators)
5. See if you're able to go to any of these individual [error pages](https://gyr-review-app-4613-162a6ccee5f3.herokuapp.com/en/hub/errors/19)

## Screenshots (for visual changes)
**- Before**
<img width="998" alt="Screenshot 2024-06-27 at 11 17 30 AM" src="https://github.com/codeforamerica/vita-min/assets/49880002/79df8d2b-da93-4077-ae47-002a4d0133a0">

**- After**
<img width="984" alt="Screenshot 2024-06-27 at 11 17 48 AM" src="https://github.com/codeforamerica/vita-min/assets/49880002/192ed660-d2c3-4e09-95d8-81dccb9eb70b">
